k8s/aat/money-claims/claim-store/claim-store.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: cmc-claim-store
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:latest-*
+spec:
+  releaseName: cmc-claim-store
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: cmc-claim-store
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "4d7922fc-7d37-4278-b542-6ac8fe0408e2"
+      image: hmcts.azurecr.io/hmcts/cmc-claim-store:latest-920f2e47ce037b5b
+      ingressHost: cmc-claim-store.service.core-compute-aat.internal
+      keyVaults:
+        cmc:
+          resourceGroup: cmc
+          secrets:
+          - claim-store-db-password
```

Comment:
@timja we have 2 DBs? should we use just one and add a new shared pw? or keep it aligned with prod 🤔

Reply:
not actually sure what plan is with dbs...where are they located?